draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1066,182 @@ The following describes each member of the `concise-bom-tag` map.
+
+Each entry in the measurement-values-map is a separate piece of evidence describing the environment named by the environment-map. An attestor can provide multiple environment-maps each containing a single values measurement-values map, a single environment-map containing multiple entries in its measurement-values-map, or a combination of these approaches.
+
+If evidence from different sources has the same environment-map then the measurement-values-maps are merged. If both measurement-value-maps being merged contain the same key then the values associated with that key MUST be binary identical. 
+
+### Transforming SPDM Evidence to a format usable for matching
+
+***Evidence Binding For SPDM*** describes the process by which evidence in a SPDM MEASUREMENTS response is converted to Evidence suitable for matching using the rules below. The converted evidence is held in evidence triples which have a similar format to reference-triples (their semantics is described below).
+
+### Transforming DICE Evidence to a format usable for matching
+
+DICE Evidence appears in certificates in the TcbInfo or MultiTcbInfo extension. Each TcbInfo, and each entry in the MultiTcbInfo, is converted to an evidence triple using the rules in this section. In a MultiTcbInfo each entry in the sequence is treated as independent and translated into a separate evidence object.
+
+The verifier SHALL translate each field in the TcbInfo into a field in the created endorsed-triple-record
+
+- The TcbInfo `type` field SHALL be copied to the field named `environment-map / class / class-id`
+- The TcbInfo `vendor` field SHALL be copied to the field named `environment-map / class / vendor`
+- The TcbInfo `model` field SHALL be copied to the field named `environment-map / class / model`
+- The TcbInfo `layer` field SHALL be copied to the field named `environment-map / class / layer`
+- The TcbInfo `index` field SHALL be copied to the field named `environment-map / class / index`
+
+- The TcbInfo `version` field SHALL be translated to the field named `measurement-map / mval / version / version`
+- The TcbInfo `svn` field SHALL be copied to the field named `measurement-map / mval / svn`
+- The TcbInfo `fwids` field SHALL be translated to the field named `measurement-map / mval / digests`
+  - Each digest within fwids is translated to a CoMID digest object, with an appropriate algorithm identifier
+- The TcbInfo `flags` field SHALL be translated to the field named `measurement-map / mval / flags`
+  - Each flag is translated independently 
+- The TcbInfo `vendorInfo` SHALL shall be copied to the field named `measurement-map / mval / raw-value`
+
+If there are multiple evidence triples with the same environment map then ...
+
+## Evidence appraisal
+
```

Comment:
I think the statement on Line 1186 is not accurate. 
In the Evidence Appraisal phase, a CoRIM Appraisal Context and an Evidence Appraisal Policy are used by the Verifier to appraise the received evidence to deduce the trustworthiness of the received Evidence.